packages/api/src/magic.link.js

```diff
@@ -1,3 +1,5 @@
+import { magicTestModeIsEnabledFromEnv, maybeJsonParseable } from './utils/env.js'
+
 export const createMagicTestTokenBypass = (
```

Comment:
If we can remove the `export` here and just export the concrete value `magicLinkBypassForUnitTestingWithTestToken`, that seems a bit cleaner. But if you need to export this for testing, that's okay by me...

Reply:
https://github.com/web3-storage/web3.storage/pull/1754/commits/88a488d3affb2f90a77918b324656798932c081b